### ISSUE TITLE
Is there a way to remove player collision?

### ISSUE THREAD

**Issue by tomster-velo** [haxball/haxball-issues#1613]
I'm helping in the making of some f1 races but i'm new in haxball programming so i wanted to know if there is a way to remove player collision during a race.

**Comment by thenorthstar** [comment]
> I'm helping in the making of some f1 races but i'm new in haxball programming so i wanted to know if there is a way to remove player collision during a race.

@VINNY-SANTORINI Yes, it's possible. What you should do is firstly set the map sensitive to changes in collision status. What I meant is both the **cGroups** **red**, **blue**, **c0** and **c1** should be in. If you want to players to collide (this is the default status in the same time), **red** and **blue** have to be in, else **c0** and **c1** have to be in.

Don't worry because I have example scripts for you. Here you can see how to create/edit your map: https://github.com/thenorthstar/HaxBall-Public-Yellow-Bot/blob/main/js/v3.js (Look at maps)

And this one is the collision script: https://github.com/thenorthstar/HaxBall-Example-Scripts/blob/main/Beginner/Collision.js

And here's the output: https://thehax.pl/forum/powtorki.php?nagranie=6d277d31ebd165373341a5627c45a4fa

**Comment by tomster-velo** [comment]
> > I'm helping in the making of some f1 races but i'm new in haxball programming so i wanted to know if there is a way to remove player collision during a race.
> 
> @VINNY-SANTORINI Yes, it's possible. What you should do is firstly set the map sensitive to changes in collision status. What I meant is both the **cGroups** **red**, **blue**, **c0** and **c1** should be in. If you want to players to collide (this is the default status in the same time), **red** and **blue** have to be in, else **c0** and **c1** have to be in.
> 
> Don't worry because I have example scripts for you. Here you can see how to create/edit your map: https://github.com/thenorthstar/HaxBall-Public-Yellow-Bot/blob/main/js/v3.js (Look at maps)
> 
> And this one is the collision script: https://github.com/thenorthstar/HaxBall-Example-Scripts/blob/main/Beginner/Collision.js
> 
> And here's the output: https://thehax.pl/forum/powtorki.php?nagranie=6d277d31ebd165373341a5627c45a4fa

thank you! where can i learn the basics of haxball programming? i get lost with room attributes

**Comment by thenorthstar** [comment]
> > > I'm helping in the making of some f1 races but i'm new in haxball programming so i wanted to know if there is a way to remove player collision during a race.
> > 
> > 
> > @VINNY-SANTORINI Yes, it's possible. What you should do is firstly set the map sensitive to changes in collision status. What I meant is both the **cGroups** **red**, **blue**, **c0** and **c1** should be in. If you want to players to collide (this is the default status in the same time), **red** and **blue** have to be in, else **c0** and **c1** have to be in.
> > Don't worry because I have example scripts for you. Here you can see how to create/edit your map: https://github.com/thenorthstar/HaxBall-Public-Yellow-Bot/blob/main/js/v3.js (Look at maps)
> > And this one is the collision script: https://github.com/thenorthstar/HaxBall-Example-Scripts/blob/main/Beginner/Collision.js
> > And here's the output: https://thehax.pl/forum/powtorki.php?nagranie=6d277d31ebd165373341a5627c45a4fa
> 
> thank you! where can i learn the basics of haxball programming? i get lost with room attributes

@VINNY-SANTORINI You can find and learn HaxBall basics on the following link: https://github.com/haxball/haxball-issues/wiki/Headless-Host

And you have to do a lot of practice to improve yourself.
